node/mock-toolrunner.ts

```diff
@@ -222,6 +222,10 @@ export class ToolRunner extends events.EventEmitter {
 
         // TODO: filter process.env
         var res = mock.getResponse('exec', cmdString);
+        if (!res) {
+            // exec call not mocked correctly?
+            this._debug(`'exec' call not mocked: ${cmdString}`);
+        }
```

Comment:
This makes me wonder whether this should be baked into `mock.getResponse()` instead. That seems more appropriate to me and I wouldn't think it would introduce any problem.